src/Console/NotifyCommand.php

```diff
@@ -59,21 +66,32 @@ public function __construct(SettingsRepositoryInterface $settings, NotificationS
     public function handle()
     {
         $days = (int) $this->settings->get('fof-best-answer.select_best_answer_reminder_days');
-        $time = Carbon::today()->addDay(-$days);
+        $time = Carbon::now()->addDay(-$days);
```

Comment:
Should use the subDays method with a positive argument.

Reply:
Change implemented :)